site/_docs/history.md

```diff
@@ -51,18 +127,280 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5946">CALCITE-5946</a>]
+ `TimeString` should allow fractional seconds ending in zero
+* In `Puffin`, allow an action to test whether it is looking at the last line of a source
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5906">CALCITE-5906</a>]
+ JDBC adapter should generate `TABLESAMPLE`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5895">CALCITE-5895</a>]
+ `TABLESAMPLE (0)` should return no rows
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5813">CALCITE-5813</a>]
+ Type inference for sql functions `REPEAT`, `SPACE`, `XML_TRANSFORM`, and `XML_EXTRACT` is incorrect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5908">CALCITE-5908</a>]
+ Refactor: Remove unnecessary null checks in `CalciteSchema`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5843">CALCITE-5843</a>]
+ Constant expression with nested casts causes a compiler crash
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5885">CALCITE-5885</a>]
+ `SqlNode#toSqlString()` does not honor dialect's `supportsCharSet()` flag on nested types
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5869">CALCITE-5869</a>]
+ `LEAST_RESTRICTIVE` does not use inner type of `MEASURE` for comparisons
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5903">CALCITE-5903</a>]
+ `RelMdCollation` does not define collations for `EnumerableLimit`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5882">CALCITE-5882</a>]
+ Compile-time evaluation of `SPLIT` function returns incorrect result
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5879">CALCITE-5879</a>]
+ AssertionError during constant reduction of `SPLIT` expression that returns null
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5875">CALCITE-5875</a>]
+ Remove unnecessary null checks in Redis adapter
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5859">CALCITE-5859</a>]
+ Compile-time evaluation of `LEFT(NULL, n)` should not throw RuntimeException
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5837">CALCITE-5837</a>]
+ `RexUtil#pullFactors` output's order should be deterministic even when the RexNode kind is `OR`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5877">CALCITE-5877</a>]
+ AssertionError during `MOD` operation if result scale is greater than maximum numeric scale
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5841">CALCITE-5841</a>]
+ Improve singleton implementation for `ChinookAvaticaServer` in calcite-plus
+* Following [<a href="https://issues.apache.org/jira/browse/CALCITE-5688">CALCITE-5688</a>]
+ Eliminate warnings in server parser
+
 #### Build and test suite
 {: #build-1-36-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5921">CALCITE-5921</a>]
+ `SqlOperatorFixture.checkFails` and `checkAggFails` don't check runtime failure (follow-up)
```

Comment:
minor: no need to mention this "follow-up", it's enough to mention this ticket only once (on the next line).